lib/core/DocsLayout.js

```diff
@@ -54,7 +55,12 @@ class DocsLayout extends React.Component {
               {metadata.previous_id && (
                 <a
                   className="docs-prev button"
-                  href={metadata.previous_id + '.html'}>
+                  href={
+                    path
```

Comment:
Please extract this path transformation code out into a function somewhere, perhaps into utils.